clcache.py

```diff
@@ -313,26 +313,17 @@ def section(self, key):
     def sections(self):
         return (CompilerArtifactsSection(path) for path in childDirectories(self._compilerArtifactsRootDir))
 
-    def removeObjects(self, stats, removedObjects):
-        for o in removedObjects:
-            dirPath = self.section(o).cacheEntryDir(o)
-            if not os.path.exists(dirPath):
-                continue  # May be if object already evicted.
-            objectPath = os.path.join(dirPath, "object")
-            if os.path.exists(objectPath):
-                # May be absent if this if cached compiler
-                # output (for preprocess-only).
-                fileStat = os.stat(objectPath)
-                stats.unregisterCacheEntry(fileStat.st_size)
```

Comment:
removeEntry is only used in clean() and in Cache.clean() the number of entries as well as the cache size is recalculated.